### PR TITLE
S2t2 get account by balance

### PR DESCRIPTION
**Pull request by avocadobelly**
A user can now see all accounts in the db and search by a specified amount

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -18,6 +18,30 @@ var getAllAccountsData = (db, callback) => {
...
 }
 
+var getAccountsBelowBal = (db, upperBal, callback) => {
+    var collection = db.collection('accounts')
+    collection.find({balance: {$lt: upperBal}}).toArray((err, docs) => {
+        results = []
+        docs.forEach(doc => {
+            doc.balance = doc.balance.toString()
+            results.push(doc)
+        })
```
Comment (sudokufan) [inline index.js]: I'm not sure you need these? My code doesn't have any of these lines, it just goes straight to 'callback(docs)'

File: index.js [hunk 2]
```diff
@@ -18,6 +18,30 @@ var getAllAccountsData = (db, callback) => {
...
+}
+
+var getAccountsAboveBal = (db, lowerBal, callback) => {
+    var collection = db.collection('accounts')
+    collection.find({balance: {$gt: lowerBal}}).toArray((err, docs) => {
+        results = []
+        docs.forEach(doc => {
+            doc.balance = doc.balance.toString()
+            results.push(doc)
+        })
```
Comment (sudokufan) [inline index.js]: same with these five

File: index.js [hunk 3]
```diff
@@ -18,6 +18,30 @@ var getAllAccountsData = (db, callback) => {
...
+        callback(results)
```
Comment (sudokufan) [inline index.js]:
```suggestion
        callback(docs)
```

File: index.js [hunk 4]
```diff
@@ -18,6 +18,30 @@ var getAllAccountsData = (db, callback) => {
...
 
+var getAccountsBelowBal = (db, upperBal, callback) => {
+    var collection = db.collection('accounts')
+    collection.find({balance: {$lt: upperBal}}).toArray((err, docs) => {
+        results = []
+        docs.forEach(doc => {
+            doc.balance = doc.balance.toString()
+            results.push(doc)
+        })
+        callback(results)
```
Comment (sudokufan) [inline index.js]:
```suggestion
        callback(docs)
```

File: index.js (hunk 1 above)
Comment (avocadobelly) [reply index.js]: This was a fix to stop Postman app treating the decimal like it was an object

File: index.js (hunk 3 above)
Comment (avocadobelly) [reply index.js]: Makes more sense to leave it as it is if I'm keeping the foreach from above

**Pull request merged**